packages/ui/src/components/pagination/acvPagination.ts

```diff
@@ -0,0 +1,29 @@
+export interface AcvPaginationProps {
```

Comment:
I would define modelValue via defineModel

Reply:
<img width="495" alt="image" src="https://github.com/user-attachments/assets/edae1e09-b78f-48ab-8f1a-2b3542c77efa">

Reply:
what I'm doing wrong?

Reply:
if you want to use defineModel, just declare it
const model = defineModel<number>(), dont try to provide all props
but in that particular case using defineProps with events is fine,
i don't see any benefits of defineModel here